tsconfig.json

```
@@ -1,9 +1,7 @@
 {
   "compilerOptions": {
-    "outDir": "lib",
     "target": "ESNext",
-    "module": "CommonJS",
-    "moduleResolution": "node",
+    "moduleResolution": "NodeNext",
     "lib": ["ES2021"],
```

We could bump this since we shouldn't be using [node.js 16](https://nodejs.dev/en/about/releases/) anymore.
I think we could go for `ES2023` if we're using target `ESNext` anyway.

```suggestion
    "lib": ["ES2023"],
```